components/filter/filter.js

```diff
@@ -356,16 +354,10 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		}
 	}
 
-	_formatFilterCount(count, max) {
```

Comment:
This function will go away once we are using the count-badge, which will handle the formatting and max value truncating for us